src/Illuminate/Cookie/Middleware/EncryptCookies.php

```diff
@@ -30,6 +37,17 @@ public function __construct(EncrypterContract $encrypter)
     }
 
     /**
+     * Disable encryption for the given cookie name(s).
+     *
+     * @param string|array $cookieName
+     * @return void
+     */
+    public static function disableFor($cookieName)
+    {
+        static::$disabled[] = array_merge(static::$disabled, (array) $cookieName);
```

Comment:
Thanks for the heads-up. One late-night refactoring too much. I sent a PR fixing this.